src/core/i18n/translate.js

```diff
@@ -14,6 +19,23 @@ export default function translate(options = {}) {
       constructor(props, context) {
         super(props, context);
         this.i18n = context.i18n;
+
+        if (this.i18n) {
```

Comment:
Can this move to server/base and client/base? Strictly speaking a component doesn't need to use `translate()` and it looks like this code might run more than once and keep configuring moment.

Changing this to `this.i18n && !this.i18n.momet` might fix that but moving it to where `i18n` is created probably makes the most sense.